torch/_prims/__init__.py

```diff
@@ -234,7 +234,7 @@ def _assert_nvfuser_op_exists(fname: str):
     try:
         from torch._C._nvfuser import FusionDefinition as fd  # type: ignore[import]
 
-        assert getattr(fd.Ops, fname)
+        assert getattr(fd.Operators, fname)
```

Comment:
`fd.ops` is actually an instance of `Operators`.  Previously, the `Ops` were defined as static methods of a class called `Ops`.  I renamed it `Operators`.  Therefore, an instance data member was not needed.  When I added the Record recording.  I needed the `Operators` class to be stateful to hold a pointer back to the original `FusionDefinition` in order to add a Record per operation and an Output State Record to the `FusionDefinition`.

So, it is proper to query the class `Operators` as to whether a given function exists rather than the instance `ops`.